src/style.css

```diff
@@ -1,3 +1,450 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
   box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+}
+
+body {
+  background: #181818;
+  height: 100vh;
+}
+
+header {
+  background-color: #383838;
+  height: 7vh;
+}
+
+.main-menu {
+  width: 30px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-menu1 {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+  margin-left: 20px;
+  margin-top: 3px;
+}
+
+.youtube-logo {
+  max-width: 120px;
+}
+
+.nav-menu2 {
+  display: flex;
+  justify-content: center;
+}
+
+.search-input {
+  padding: 0 200px 0 150px;
+  margin: 5px 0 0 0;
+  background-color: #181818;
+  border: 1px;
+  margin-right: 13px;
+}
+
+input::placeholder {
+  position: absolute;
+  top: 10px;
+  left: 13px;
+  color: rgba(255, 255, 255, 0.685);
+  font-weight: 400;
+  line-height: 10px;
+  font-size: 150%;
+  padding-bottom: 20px;
+}
+
+.video-icon {
+  position: relative;
+  top: 5px;
+  right: 13px;
+  height: 34px;
+  width: 60px;
+  background-color: rgb(75, 75, 75);
+  border-radius: 2px;
+}
+
+.video-search {
+  position: relative;
+  top: 5px;
+  left: 15px;
+  max-width: 100%;
+  text-align: center;
+}
+
+.microphone {
+  position: relative;
+  right: 8px;
+  top: 1px;
+  max-width: 4%;
+}
+
+.nav-menu3 {
+  display: flex;
+  justify-items: center;
+  gap: 12px;
+  margin-right: 50px;
+}
+
+.fa-bell-plus,
+.fa-video-plus {
+  position: relative;
+  top: 8px;
+  align-items: center;
+  color: white;
+  font-size: 22px;
+  margin-left: 5px;
+}
+
+.fa-video-plus {
+  margin-left: 5px;
+}
+
+.user-img {
+  position: relative;
+  top: 6px;
+  height: 1.6rem;
+  width: 1.6rem;
+  border-radius: 50%;
+  margin-left: 5px;
+}
+
+.content-body {
+  display: flex;
+  height: 90vh;
+  margin: 20px 10px 90px 90px;
+}
+
+.main-body {
+  width: 100%;
+  flex-shrink: 1;
+}
+
+.sab-video {
+  width: 100%;
```

Comment:
- [ ] Avoid code repetion, you have this width: 100%; 3 times, please fix this, you can use a class and apply that class everywhere this property is needed